build.js

```diff
@@ -4,31 +4,33 @@ import { fileURLToPath } from 'url';
 
 import { build } from 'esbuild';
 
+const [, , logLevel = 'info'] = process.argv;
 const pkg = JSON.parse(await readFile(new URL('package.json', import.meta.url)));
 
 await build({
   entryPoints: ['src/index.ts', 'src/tailwindcss.worker.ts'],
   bundle: true,
   external: Object.keys({ ...pkg.dependencies, ...pkg.peerDependencies }),
-  logLevel: 'info',
+  logLevel,
   outdir: '.',
   sourcemap: true,
   format: 'esm',
   target: ['es2020'],
   define: {
     'process.env.DEBUG': 'undefined',
+    'process.env.JEST_WORKER_ID': 'undefined',
     __dirname: '"/"',
```

Comment:
Who needs it this time?

Reply:
https://github.com/tailwindlabs/tailwindcss/blob/master/src/featureFlags.js#L38